src/DependencyInjection/EventBusExtension.php

```diff
@@ -37,6 +38,10 @@ protected function loadInternal(array $mergedConfig, ContainerBuilder $container
 
         if ($mergedConfig['logging']['enabled']) {
             $loader->load('event_bus_logging.yml');
+
+            $container->getDefinition('simple_bus.event_bus.notifies_message_subscribers_middleware')
+                ->addArgument(new Reference('logger'))
+                ->addArgument('%simple_bus.event_bus.logging.level%');
```

Comment:
I would add empty arguments in the service config in `event_bus.yml`. Otherwise, you will get weird results when additional arguments are added to it in the future.